libr/core/windows_heap.c

```diff
@@ -191,26 +226,62 @@ static void free_extra_info(PDEBUG_HEAP_INFORMATION heap) {
 	}
 }
 
-static bool DecodeHeapEntry(PHEAP heap, PHEAP_ENTRY entry) {
-#if defined(_M_X64)
-	(WPARAM)entry += sizeof (PVOID);
-#endif
+static WPARAM GetNtDllOffset(RDebug *dbg) {
+	RList *map_list = r_w32_dbg_maps (dbg);
```

Comment:
Can this list be null? Maybe better to rreturn to assert this

Reply:
If `map_list` is null `r_list_foreach` won't do anything and the function will return 0, so it shouldn't be necessary.